doc/CONTRIBUTING.md

```diff
@@ -106,6 +108,13 @@ Don't worry about setting DynamoDB endpoints or creating tables: the broker's co
 
 **Note:** The local version of DynamoDB is not recommended for production.
 
+### Install RabbitMQ
+
+RabbitMQ is used to pass jobs to the validator, and requires Erlang to be installed before RabbitMQ.
```

Comment:
Do we need to put a note here that this won't work on anything higher than Windows 7? Or should that note go somewhere closer to the top of the install directions, so those folks don't get halfway through only to be foiled by RabbitMQ?